metallb/start_frr.sh

```diff
@@ -1,37 +0,0 @@
-#!/usr/bin/bash
-
-metallb_dir="$(dirname $(readlink -f $0))"
-
-source ${metallb_dir}/../common.sh
-source ${metallb_dir}/../network.sh
-
-## kill frr container if running
-set +e
-sudo podman inspect frr 2>/dev/null > /dev/null
-if [[ $? -eq 0 ]]; then
-	sudo podman kill frr
-fi
-set -e
-
-## configure frr
-# cleanup frr directory
-[[ -d ${metallb_dir}/frr ]] &&
-	sudo rm -rf ${metallb_dir}/frr
-
-# add firewalld rules
-sudo firewall-cmd --zone=libvirt --permanent --add-port=179/tcp
-sudo firewall-cmd --zone=libvirt --add-port=179/tcp
```

Comment:
will it work without this (and port 180 for the other container)?

Reply:
you are right, we need that, i will add it